scripts/git_utils.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
1. This script is sourced in scripts for the POSIX shell and doesn't contain anything bash-specific. I believe the shebang should be for the POSIX shell.
2. Even if we keep it for bash, it's not a good idea to force bash from `/bin` as it's ancient on macOS.

The overall suggestion is 

```suggestion
#!/usr/bin/env sh
```